tests/door43_tools_tests/test_templaters.py

```diff
@@ -148,7 +151,9 @@ def verifyBibleTemplater(self, success, expect_success, output_folder, files_to_
             file_name = os.path.join(output_folder, file_to_verify)
             self.assertTrue(os.path.isfile(file_name), 'file not found: {0}'.format(file_name))
 
-    def verifyTaTemplater(self, success, expect_success, output_folder, filesToVerify=[]):
+    def verifyTaTemplater(self, success, expect_success, output_folder, filesToVerify=None):
+        if not filesToVerify:
+            filesToVerify = None
```

Comment:
@richmahn - this doesn't make sense.  Is it supposed to be like line 128?

Reply:
Not suppose to have mutatable structures in the definition.

Reply:
Not suppose to have mutatable structures in the definition.